tests/gtest/aviftest_helpers.cc

```diff
@@ -13,13 +13,39 @@ namespace testutil {
 namespace {
 
 constexpr int AVIF_CHAN_A = AVIF_CHAN_V + 1;
-constexpr int AVIF_CHANS[] = {AVIF_CHAN_Y, AVIF_CHAN_U, AVIF_CHAN_V,
-                              AVIF_CHAN_A};
 
 }  // namespace
 
 //------------------------------------------------------------------------------
 
+AvifRgbImage::AvifRgbImage(const avifImage* yuv, int rgbDepth,
+                           avifRGBFormat rgbFormat) {
+  avifRGBImageSetDefaults(this, yuv);
+  depth = rgbDepth;
+  format = rgbFormat;
+  avifRGBImageAllocatePixels(this);
+}
+
+//------------------------------------------------------------------------------
+
+RgbChannelOffsets GetRgbChannelOffsets(avifRGBFormat format) {
```

Comment:
Optional: This function could return a fixed value for `a`, such as 0, for formats that do not have an alpha channel.